client/landing/subscriptions/hooks/use-subscribers-filter-options.ts

```diff
@@ -0,0 +1,19 @@
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { SubscribersFilterBy } from 'calypso/my-sites/subscribers/constants';
+
+const FilterBy = SubscribersFilterBy;
+
+const useSubscribersFilterOptions = () => {
+	const translate = useTranslate();
+	return useMemo(
+		() => [
+			{ value: FilterBy.All, label: translate( 'All' ) },
+			{ value: FilterBy.Email, label: translate( 'Email' ) },
+			{ value: FilterBy.WPCOM, label: translate( 'WPCOM' ) },
```

Comment:
`WPCOM` doesn't seems to be a user friendly name, maybe renaming to `WordPress.com Account`?

Reply:
Just linking here related discussion: p1688455815463729/1688056130.387729-slack-C02TCEHP3HA.

Reply:
In fact, seems like the distinction is between `Email subscriber` and `Follower` p1688056130387729-slack-C02TCEHP3HA
I'll implement it like that.

Reply:
As far as I know, we are trying to avoid the word `Followers` to avoid confusion (p1688136324494139/1688056130.387729-slack-C02TCEHP3HA).